homework-submission/homework-week1/homework-week2/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <style>
+
+    .listItem{
+    line-height: 2;
+    font-family: Roboto,sans-serif;
+    font-size: 14px;
+    margin: 3px;
+    text-align: center;
+    font-weight: 300;
+    background-color: #FFFFFF;
+    box-sizing: border-box;
+    width: 100%;
+    height: 30px;
+    box-shadow: 1px 1px 5px #d8d0d0;
+    }
+
+    .listItem:hover{
+    background-color: tomato;
+    box-shadow: 1px 1px 10px #9e9999;
+    }
+
+    .addItem:hover{
+    background-color: lightgreen;
+    }
+
+    input[value] {
+    color: #9e9999;
+    }
+
+    .list {
+    margin: auto;
+    list-style-type: none;
+    background-color: rgb(245, 245, 245);
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    width: 100%;
+    padding: 10px;
+    text-align: center;
+    }
+
+    .wrapper {
+    font-family: Roboto,sans-serif;
+    font-size: 14px;
+    margin: auto;
+    background-color: rgb(245, 245, 245);
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    width: 30%;
+    padding: 10px;
+    text-align: center;
+    }
```

Comment:
maybe you can use a separate file for the styles and import it here in the index?